README.md

```diff
@@ -1,9 +1,13 @@
-## AutoGluon: AutoML Toolkit for Deep Learning
+
+
+![AutoGluon](https://user-images.githubusercontent.com/16392542/77208906-224aa500-6aba-11ea-96bd-e81806074030.png)
```

Comment:
You may consider replacing this: https://github.com/awslabs/autogluon/blob/master/docs/static/gluon-logo.svg

Reply:
Also https://github.com/awslabs/autogluon/blob/master/docs/static/favicon.png

Reply:
Good points, I will likely leave those for a separate PR

Reply:
Why not replace them all now?

Reply:
Because I don't have the favicon version of the logo and I'd have to hack it myself, plus I'm not sure if the logo would look right on first try, whereas I can quickly confirm for the readme. Easier to do in a separate PR

Reply:
for readme file why not use relative path `./docs/static/gluon-logo.svg` ?

Reply:
We could after its replaced. If you look at many other packages though, they use an online link for their readme as I am doing (example: TensorFlow)

Reply:
> Because I don't have the favicon version of the logo and I'd have to hack it myself, plus I'm not sure if the logo would look right on first try, whereas I can quickly confirm for the readme. Easier to do in a separate PR

Makes sense, didn't realize we don't have favicon yet.
I think online link is fine for readme file, as I see this in many repos also.